cms/djangoapps/contentstore/views/tests/test_user.py

```diff
@@ -315,3 +318,11 @@ def assert_enrolled(self):
             CourseEnrollment.is_enrolled(self.ext_user, self.course.id),
             'User ext_user should have been enrolled in the course'
         )
+
+    def test_invalid_course_id(self):
+        """ Asserts that Http404 is raised when the course id is not valid. """
```

Comment:
Why are we writing test for twice ? 

If we need the test twice, should we create some common method/class that these tests can inherit?